lms/lmsdb/models.py

```diff
@@ -298,15 +344,38 @@ class Exercise(BaseModel):
     due_date = DateTimeField(null=True)
     notebook_num = IntegerField(default=0)
     order = IntegerField(default=0, index=True)
+    course = ForeignKeyField(Course, backref='exercise')
+    number = IntegerField()
+
+    class Meta:
+        indexes = (
+            (('course_id', 'number'), True),
+        )
```

Comment:
We should add this to the migration

Reply:
This migration would be solved after [this](https://github.com/PythonFreeCourse/lms/pull/307#discussion_r716048276) this migration